gettsim/transfers/wohngeld.py

```diff
@@ -513,9 +529,9 @@ def wohngeld_vor_vermög_check_m_hh(
     if haushaltsgröße_hh > max_considered_hh_größe:
```

Comment:
We should rename `max_considered_hh_größe` to `max_berücks_personen ` and base it on:

```
wohngeld_params["bonus_sehr_große_haushalte"][
        "max_anz_personen_normale_berechnung"
    ]